store/src/java/com/zimbra/cs/account/auth/AuthContext.java

```diff
@@ -73,6 +73,26 @@
      */
```

Comment:
Please import HashMap and Map after package declaration , like this-
import java.util.Map;
import java.util.HashMap